salt/metalk8s/kubernetes/CNI/calico/configured.sls

```diff
@@ -1,8 +1,8 @@
 {%- from "metalk8s/map.jinja" import kube_api with context %}
-{%- from "metalk8s/map.jinja" import defaults with context %}
+{%- from "metalk8s/map.jinja" import kubernetes with context %}
 
 include:
-  - metalk8s.kubeadm.init.certs.installed
+  - metalk8s._internal.certs
```

Comment:
Given this, I'd rename that state to `m2crypto` or something, to make it clear what it's actually about.